.circleci/config.yml

```diff
@@ -21,15 +21,14 @@ references:
   _install-requirements: &install-requirements
     run:
       name: Install Requirements
-      command: pipenv install
+      command: pipenv install --dev
```

Comment:
Should there be two install requirements:
- install requirements > pipenv install
- install dev requirements > pipenv install --dev

Reply:
We dont need 2 different install requirements - the prod install is done on the docker image meaning we can install dev requirements in the build with no ill effect